From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matthias Riechmann <21068765+mmriech@users.noreply.github.com>
Date: Tue, 21 Jan 2020 14:57:45 +0100
Subject: [ARCHEOLOGY] Fixed makefile of kernel headers 4.4.210 for rk3399
 (#1753)

> X-Git-Archeology: - Revision e0b17387ea6614340a58c75ed111d13141cb5886: https://github.com/armbian/build/commit/e0b17387ea6614340a58c75ed111d13141cb5886
> X-Git-Archeology:   Date: Tue, 21 Jan 2020 14:57:45 +0100
> X-Git-Archeology:   From: Matthias Riechmann <21068765+mmriech@users.noreply.github.com>
> X-Git-Archeology:   Subject: Fixed makefile of kernel headers 4.4.210 for rk3399 (#1753)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 23604e8a0dcdf81ec6c28ccd4b2a64b90816d8e7: https://github.com/armbian/build/commit/23604e8a0dcdf81ec6c28ccd4b2a64b90816d8e7
> X-Git-Archeology:   Date: Fri, 19 Jun 2020 17:27:27 +0200
> X-Git-Archeology:   From: Paolo <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: Introducing Rockchip RK322X SoC support (#2032)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
---
 arch/arm64/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/Makefile b/arch/arm64/Makefile
index 197c08ee6af6..d36b42ca1e51 100644
--- a/arch/arm64/Makefile
+++ b/arch/arm64/Makefile
@@ -141,6 +141,7 @@ archclean:
 	$(Q)$(MAKE) $(clean)=$(boot)
 	$(Q)$(MAKE) $(clean)=$(boot)/dts
 
+ifeq ($(KBUILD_EXTMOD),)
 # We need to generate vdso-offsets.h before compiling certain files in kernel/.
 # In order to do that, we should use the archprepare target, but we can't since
 # asm-offsets.h is included in some files used to generate vdso-offsets.h, and
@@ -150,6 +151,7 @@ archclean:
 prepare: vdso_prepare
 vdso_prepare: prepare0
 	$(Q)$(MAKE) $(build)=arch/arm64/kernel/vdso include/generated/vdso-offsets.h
+endif
 
 define archhelp
   echo  '* Image.gz      - Compressed kernel image (arch/$(ARCH)/boot/Image.gz)'
-- 
Armbian

